EIPS/eip-versioning-scheme-for-eips.md

```diff
@@ -0,0 +1,107 @@
+---
+eip: <to be assigned>
```

Comment:
```suggestion
eip: 7577
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.